phone/src/locale/en.json

```
@@ -115,6 +115,8 @@
     "APPS_MESSAGES_MESSAGE_GROUP_DUPLICATE": "This message group already exists",
     "APPS_MESSAGES_MESSAGE_GROUP_CREATE_FAILED": "Failed to create message group",
     "APPS_MESSAGES_MESSAGE_GROUP_CREATE_MINE": "You cannot add yourself!",
+    "APPS_MESSAGES_MESSAGE_GROUP_CREATE_ALL_NUMBERS_FAILED": "None of the phone numbers could be found!",
```

The wording of this is a bit weird when its a singular phone number. 

```suggestion
    "APPS_MESSAGES_MESSAGE_GROUP_CREATE_ALL_NUMBERS_FAILED": "Invalid phone number(s).",
```

It may be a good idea to use `"APPS_MESSAGES_MESSAGE_GROUP_CREATE_SOME_NUMBERS_FAILED"` and just generalize it too. (See below:)